azerothcore/azerothcore-wotlk
# Fix (DB\Skin Loot) Remove Skinning loot  from none skinnable npc

Removes skinning loot and fixes flag from a npc that was never skinnable.
https://classic.wowhead.com/npc=15204/high-marshal-whirlaxis
![image](https://user-images.githubusercontent.com/16887899/147276057-80d5c820-d08f-4360-997d-20b7ce1086e2.png)



## Changes Proposed:
-  Removes skinning loot and fixes flag from a npc that was never skinnable.

## Issues Addressed:

- Closes: https://github.com/chromiecraft/chromiecraft/issues/2661
- Closes: https://github.com/azerothcore/azerothcore-wotlk/issues/9863

## SOURCE:


## Tests Performed:

- Creature does not give Skinning option that never worked in game.


## How to Test the Changes:


1. goto any of the entries of the npc or just do a .npc add 15204
2. murder\violate it and then attempt to skin
3. notice it does not give skinning icon or option anymore as intended.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
